styles/main.css

```diff
@@ -0,0 +1,7 @@
+img {
+    height: 30px;
+}
+
+.titleError, .authorError, emailError, .phoneError, .isbnError {
+    display: none;
```

Comment:
add styling, display none will not render anything there on div where error needs to show some text, use appropriate styling for highlighting the text by testing it on UI then mark it done